config/webpack/webpack.dev.js

```diff
@@ -24,4 +24,5 @@ module.exports = merge(common, {
         disableHostCheck: true,
         stats: { children: false },
     },
+    devtool: 'inline-source-map',
```

Comment:
@kapilgorve Can you please change it to `eval-cheap-module-source-map`. It is faster then `inline-source-map`.
https://webpack.js.org/configuration/devtool/

Reply:
Changed.